### PR TITLE
Support Ubuntu Focal

### PR DESCRIPTION
**Pull request by danielsreichenbach**

### CODE REVIEW

File: spec/classes/puppet_agent_spec.rb [hunk 1]
```diff
@@ -104,7 +104,7 @@
 
         it { is_expected.to contain_class('puppet::agent::service::systemd').with_enabled(false) }
         case os
-        when /\Adebian-/, /\A(redhat|centos|scientific)-7/, /\Afedora-/, /\Aubuntu-(16|18)/, /\Aarchlinux-/
+        when /\Adebian-/, /\A(redhat|centos|scientific)-7/, /\Afedora-/, /\Aubuntu-(16|18|20)/, /\Aarchlinux-/
```
Comment (ekohl) [inline spec/classes/puppet_agent_spec.rb:107]: This is all the Ubuntu versions we support. I won't repeat it for every line in this file
```suggestion
        when /\Adebian-/, /\A(redhat|centos|scientific)-7/, /\Afedora-/, /\Aubuntu-/, /\Aarchlinux-/
```

**Comment by ekohl** [comment]
Looks like it can't install puppetserver. Have those packages been released yet for Focal?

**Comment by danielsreichenbach** [comment]
> Looks like it can't install puppetserver. Have those packages been released yet for Focal?
That seems to be the case. Just assumed that was already there since they pushed out puppet-agent for Focal :sob:

**Comment by ekohl** [comment]
Given there's no code change needed for Ubuntu 20.04 agents, I'm going to keep this open until there are Puppetserver packages available.

**Comment by mmoll** [comment]
Continuation in GH-746, danke @danielsreichenbach!

**Pull request closed without merging**